### DISCUSSION TITLE
Feed parse errors megathread

### DISCUSSION THREAD

**Discussion by jocmp** [jocmp/capyreader#322] [Feature Requests]
Taking a page out of ReadYou's book. This is an ongoing thread to discuss and debug issues parsing certain feeds.

If you have a problem with a feed, post the feed link here. I can't promise that all feeds are fixable but I'll always try.

**Comment by ConEstArthur** [comment]
Should we have a discussion on improving scraping few websites on capy reader as well ?
Because there is stark difference between how Feeder scraps the article and how Capy does it.

You can check for this rss -
https://indianexpress.com/section/opinion/feed/

You can also open a particular article in a browser and check. I've found capy reader ignores certain elements and also fails to load few images.

**Reply by jocmp** [reply]
@ConEstArthur - feel free to post issues like that here. I'll take a look at this feed. Do you have any examples of what elements are ignored?

**Reply by ConEstArthur** [reply]
![image](https://github.com/user-attachments/assets/bed8e873-f76a-412a-8915-1c70381e0f8c)

It misses this part in this article - https://indianexpress.com/article/opinion/columns/tavleen-singh-writes-humility-works-better-than-arrogance-9555721/

**Comment by jocmp** [comment]
Archiving this thread since users are opening bug tickets instead. I'm still tracking the partial content issue via https://github.com/jocmp/capyreader/issues/348.
